draft-ietf-acme-acme.md

```diff
@@ -1995,9 +1995,21 @@ paths to various trust anchors. Clients can fetch these alternates and use their
 own heuristics to decide which is optimal.
 
 ~~~~~~~~~~
-GET /acme/cert/asdf HTTP/1.1
+POST /acme/cert/asdf HTTP/1.1
 Host: example.com
-Accept: application/pem-certificate-chain
+Content-Type: application/jose+json
+Accept: application/pkix-cert
```

Comment:
Is there a reason you are switching back to `application/pkix-cert`? This has just been [changed one month ago](https://github.com/ietf-wg-acme/acme/pull/442/files#diff-8430e2aa241beb4ac49b252db20d4ee8R1946) and I cannot see this mentioned in David's email.

Reply:
Copy/paste error.  This also caused me to clean up some other spurious Accept headers that had gotten copy/pasted into other examples.